crates/env/src/api.rs

```diff
@@ -175,13 +175,13 @@ where
 }
 
 /// Emits an event with the given event data.
-pub fn emit_event<E, Event>(event: Event)
+pub fn emit_event<E, Evt>(event: Evt)
```

Comment:
Nit: `Evnt` sounds more idiomatic for a generic

Reply:
I named it `Evt` as I thought it is a common abbreviation of `Event` e.g. https://github.com/abbrcode/abbreviations-in-code#e.